packages/augur-core/source/contracts/trading/ZeroXTrade.sol

```diff
@@ -20,6 +21,8 @@ contract ZeroXTrade is Initializable, IZeroXTrade, IERC1155 {
 
     bool transferFromAllowed = false;
 
+    uint256 constant public TRADE_INTERVAL_VALUE = 10 ** 19; // Trade value of 10 DAI
```

Comment:
I generally say that storage values should be public, but in the case of constants I think it just adds noise and doesn't add value.